packages/ember-engines/lib/utils/ensure-lazy-loading-hash.js

```diff
@@ -8,11 +8,15 @@
  * @return {Object}
  */
 module.exports = function ensureLazyLoadingHash(context) {
-  if (typeof context.lazyLoading === 'boolean') {
+  if (
+    typeof context.lazyLoading === 'boolean' ||
+    typeof context.lazyLoading === 'undefined'
```

Comment:
I don't think we need `typeof` here for checking undefined, we can just compare:

```suggestion
    context.lazyLoading === undefined
```